testing/amiitool/APKBUILD

```diff
@@ -0,0 +1,25 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=amiitool
+pkgver=r2
```

Comment:
Sadly this does not work as expected with apk, as it treats it as a postfix (like a pre-release). So it's best to just use `2` as the version and then prepend the 'r' where necessary.

Reply:
Roger, I already suggested upstream to switch to semver.

Will fix asap.

Reply:
You forgot to put the 'r' back in the url, it now 404's

Reply:
doh, too many balls in the air. I have made the corrections.